### ISSUE TITLE
Mob Spawners Bug

### ISSUE THREAD

**Issue by Zano1999** [goncalomb/NBTEditor#23]
When I set up a custom mob spawner with an anime book it works for a few seconds after that it becomes a spawner for pigs, please fix.

**Comment by goncalomb** [comment]
The problem may be that the spawner still contains the pig...

When you add an entity to the spawner it doesn't remove the pig. If you use `/nbts info` you will see that it contains the pig and the custom entity. To remove the pig use `/nbts del 0` (the pig is probably the first entity on the list), after that the spawner may need to trigger one more time before starting to spawn only the custom entity.

I'm testing some spawners to see if there is another issue.

**Comment by Zano1999** [comment]
Solved, thanks for the help
